### PR TITLE
feat: config query validation

### PR DESCRIPTION
**Pull request by adityathebe**
Related to https://github.com/flanksource/duty/issues/77

### CODE REVIEW

File: query.go [hunk 1]
```diff
@@ -3,11 +3,105 @@ package duty
 import (
 	"context"
 	"fmt"
+	"strings"
```
Comment (moshloop) [inline query.go:7]: Can we move this into a `config` package ?

Comment (moshloop) [reply query.go:7]: @adityathebe ^

**Comment by adityathebe** [comment]
Need to refactor. Ginkgo test suites inside `config` package isn't running.

File: config/query_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+			t.Fail()
+		}
+		if output != test.expectedOutput {
+			t.Logf("%s: validateTablesInQuery() returned %v, but expected %v", test.name, output, test.expectedOutput)
+			t.Fail()
+		}
+	}
+}
+
+// var _ = ginkgo.Describe("Query should only run non mutation queries", func() {
```
Comment (adityathebe) [inline config/query_test.go:84]: This ginkgo spec is not moved to pkg_config_test.go because we're testing an unexported function.

**Pull request merged**